parm/config/config.resources

```diff
@@ -522,8 +522,7 @@ elif [[ "${step}" = "fcst" || "${step}" = "efcs" ]]; then
         # PETS for quilting
         if [[ "${QUILTING:-}" = ".true." ]]; then
           (( QUILTPETS = ntasks_quilt * nthreads_fv3 ))
-          (( WRTTASK_PER_GROUP = WRTTASK_PER_GROUP * nthreads_fv3 ))  # when threads are used, WRTTASKS_PER_GROUP = INCOMING_WRTTASKS_PER_GROUP * threads
-          # model_configure should be updated to reflect the number of threads used by calling this variable write_pets_per_group
+          (( WRTTASK_PER_GROUP = WRTTASK_PER_GROUP ))
```

Comment:
This is incorrect.
The `WRTTASKS_PER_GROUP` need to account the threads the atmosphere is running with.

Reply:
Not anymore, apparently. I only made this change after the forecast failed due to a PE mismatch. This line is rather superfluous now though, I should've just deleted it.

Reply:
Maybe @junwang-noaa can clarify?

Reply:
What I see here is that the WRTTASKS_PER_GROUP is the multiple of the MPI write tasks per group * nthreads_fv3. In other words, inside the write grid comp, the number of MPI tasks is actually WRTTASKS_PER_GROUP/nthreads_fv3, which I think is the original WRTTASKS_PER_GROUP.

Reply:
I suggest just delete this line.  It is superfluous as you note.
There is some major confusion regarding threading in the write task calculation.
```suggestion
```